Sources/Shared/Networking/Networking.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+open class Networking {
```

Comment:
@vadymmarkov User can configure the `Networking` by assigning new one to the `AuthConfig`, so it should be either `public` or `open`